### DISCUSSION TITLE
Add watermark for PDF

### DISCUSSION THREAD

**Discussion by louiskhenghao** [pdfme/pdfme#122] [Q&A] [answered]
Hi everyone, just wan to ask Is there any easy way that we could add watermark to PDF?

Expected Result 
<img width="798" alt="CleanShot 2022-12-28 at 12 55 27@2x" src="https://user-images.githubusercontent.com/5268771/209759691-8096ca7e-ab51-4ea4-a28a-c05149230f7b.png">

Reference: https://pdfmake.github.io/docs/0.1/document-definition-object/watermark/

**Comment by hand-dot** [comment, marked as answer]
Hi @louiskhenghao !

You can imitate the watermark by creating a template dynamically.
I wouldn't create the edge case feature.

**Reply by louiskhenghao** [reply]
Thanks for your info, currently I did something like what you have just mentioned. 
The reason to ask is also to see if there were any simple ways to achieve this. 
Anyway thank you once again for creating this great library
